src/pathology-report/ReportComponent.css

```diff
@@ -1,17 +1,55 @@
 .table {
-    width: 100%;
-    position: relative;
-  }
-  
-  .tr {
-    border-bottom: 1px solid lightgrey;
-    line-height: 2rem;
-  }
-  
-  .tr:nth-child(even) {
-    background-color: var(--omrs-color-bg-medium-contrast);
-  }
-  
-  .table td {
-    padding: 0.25rem 0.5rem;
-  }
\ No newline at end of file
+  width: 100%;
+  position: relative;
+}
+.tableContainer {
+  height: 500px;
+  overflow-x: auto;
+  margin: 0 1rem;
+  border: 2px solid lightgrey;
+  border-radius: 0.25rem;
+}
+
+.tr {
```

Comment:
Be sure to delete CSS that isn't being used. It's easy for junk CSS to accumulate.